test/LendingPool.t.sol

```diff
@@ -665,20 +665,44 @@ contract DepositAndWithdrawalTest is LendingPoolTest {
         pool.donateToTranche(1, amount);
     }
 
-    function testSuccess_donateToPool(uint8 index, uint128 assets, address donator) public {
+    function testRevert_donateToPool_DonateToTranche(uint32 initialShares, uint128 assets, address donator) public {
```

Comment:
Function on leningpool is still named `donateToTranche`, was this supposed to be updated as well?